coordinator/tasks/feature.go

```diff
@@ -105,34 +109,36 @@ func (t *FeatureTask) Run() error {
 	if sourceLocationErr != nil {
 		return sourceLocationErr
 	}
-	t.logger.Debugw("Feature's source location", "location", sourceLocation, "location_type", sourceLocation.Type())
+	logger.Debugw("Feature's source location", "location", sourceLocation, "location_type", sourceLocation.Type())
```

Comment:
You could be throwing almost all these into logger.With( so that all the following logs keep context too which can be useful, not always required, your call